src/components/Create/hooks/useLoadInitialStateFromQuery.ts

```diff
@@ -73,7 +73,7 @@ const parseCreateFlowStateFromInitialState = (
   }
 
   const reconfigurationRuleSelection =
-    ballotStrategiesFn().find(s =>
+    ballotStrategiesFn({}).find(s =>
```

Comment:
This feels like a bug - is it possible to allow this to take in undefined instead?

Reply:
Fixed this sir :pray: